source/locale/en/symbols.dic

```diff
@@ -112,22 +112,18 @@ _	line	most
 ◾	black square	some
 □	white square	some
 ◦	white bullet	some
-⇒	right double arrow	some
 ⇨	right white arrow	some
 ➔	right-pointing arrow	some
 ➢	right arrowhead	some
-⮚	right arrowhead	some
 ❖	black diamond minus white X	some
 ♣	black club	some
 ♦	black diamond	some
-◆	black diamond	some
+◆	black diamond	some	

```

Comment:
It looks like this introduces a white space error (extra tab on the end of the line).